test/unit/data/events/get-events.test.js

```diff
@@ -0,0 +1,79 @@
+jest.mock('../../../../app/storage')
+const { getClient: mockGetClient, odata: mockOdata } = require('../../../../app/storage')
+
+const mockListEntities = jest.fn()
+
+const mockTableClient = {
+  listEntities: mockListEntities
+}
+
+const { PARTITION_KEY } = require('../../../mocks/values/partition-key')
+const { CATEGORY } = require('../../../mocks/values/category')
+const { PAYMENT_EVENT } = require('../../../../app/constants/event-types')
+
+const { stringifyEventData } = require('../../../helpers/stringify-event-data')
+
+let extractedEvent
+let enrichedEvent
+let events
+
+const { getEvents } = require('../../../../app/data/events/get-events')
+
+describe('get events', () => {
+  beforeEach(() => {
+    jest.clearAllMocks()
+
+    extractedEvent = JSON.parse(JSON.stringify(require('../../../mocks/events/extracted')))
+    enrichedEvent = JSON.parse(JSON.stringify(require('../../../mocks/events/enriched')))
+
+    stringifyEventData(extractedEvent)
+    stringifyEventData(enrichedEvent)
+
+    events = [extractedEvent, enrichedEvent]
+
+    mockGetClient.mockReturnValue(mockTableClient)
+    mockListEntities.mockReturnValue(events)
```

Comment:
I think these mocks should probably be just below jest.clearAllMocks()

Reply:
I would agree but events, needs all of the in between lines so can't be moved

Reply:
What do you mean exactly?

Reply:
The second mock` mockListEntities` has `events` as it's mocked return.  But events is configured through the lines between this and `jest.clearAllMocks`.  So if these move up, events will be undefined.

Reply:
Yep, don't want events to be undefined!